### DISCUSSION TITLE
"Uptime Kuma" in disable auth confirmation dialog

### DISCUSSION THREAD

**Discussion by Alanimdeo** [louislam/dockge#44] [General]
<img width="534" alt="image" src="https://github.com/louislam/dockge/assets/32806763/338246dc-f562-4ef2-86fb-fe7bb56e9e1d">

PS: I'm translating Dockge into Korean, but many strings are not contained in en.json, and some are hard-coded.

**Comment by louislam** [comment]
Yes, translation is not ready yet.
